app/helpers/patient_helper.rb

```diff
@@ -28,6 +32,29 @@ def normalize_and_get_state_name(name)
     state_names[normalize_name(name)] || nil
   end
 
+  # This function will attempt to match the input to a language in the system
+  # PARAM: `lang` can be a three-letter iso-639-2t code, a two-letter iso-639-1 code, or the name (not case sensitive)
+  # PARAM EXAMPLES: 'eng', 'en', 'English', 'ENGLISH' <-- All will map to 'eng'
+  # RETURN VALUE: `nil` if unmatchable, else the three-letter iso code ('eng')
+  def normalize_and_get_language_name(lang)
+    return nil if lang.nil?
+
+    # spa-PR is the only iso-code that involves case. it will not be properly matched if
+    # we call downcase on the input
+    lang = lang.casecmp('spa-pr')&.zero? ? 'spa-PR' : lang.to_s
+    matched_language = nil
+    matched_language = lang.to_sym if all_languages[lang.to_sym]
```

Comment:
I think here we can make `lang` lowercase. That way something like `ENG` will work, since technically our codes aren't supposed to be case sensitive.

Reply:
Good point! It should be there, i forgot to copy it over. My bad. Fixing now.